packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -93,6 +93,12 @@ export function hasDefaultSerialize(route: Route): boolean {
   return route.serialize === defaultSerialize;
 }
 
+function getOwnerChecked(route: Route): Owner {
+  let owner = getOwner(route);
+  assert('Route is unexpectedly missing an owner', owner);
+  return owner;
+}
```

Comment:
One concern: this seems good, but it looks like we'll end up shipping a bunch of extra copies of it to prod this way?

Reply:
Yeah, I could just inline the assert, it was annoying, heh.

Reply:
Inlined.